turbo.json

```diff
@@ -8,6 +8,7 @@
       "outputs": []
     },
     "dev": {
+      "dependsOn": ["@webstudio-is/sdk#build"],
```

Comment:
I think dev should not depend on build, everything that was supposed to be built in SDK, should probably be built just once upon installation, after that, SDK is just a typescript code

Reply:
In that case the `main` and `types` should be changed in `package.json` of `webstudio-is` package to:
```json
  "main": "src/index.ts",
  "types": "src/index.ts",
```

Reply:
@andrasbacsai that sounds good to me or is there a problem with doing so?